cli/utilities/containers.py

```diff
@@ -41,3 +45,81 @@ def login(
             raise ContainerRegistryError(
                 f"Failed to login to container registry '{registry}'"
             )
+
+
+class Container(Cli):
+    """This module provides CLI interface for container operations"""
+
+    def __init__(self, nodes, package="podman"):
+        super(Container, self).__init__(nodes)
+        self.base_cmd = package
+
+    def run(
+        self,
+        image=None,
+        rm=None,
+        env=None,
+        volume=None,
+        ports=None,
+        detach_key=None,
+        restart=None,
+        detach=False,
+        cmds=None,
+    ):
+        """Executes the provided command using podman
+        Args
+            image (str): Image name
+            env (list): List of environment variables
+            volume (list): List of volumes
+            ports (list): List of ports
+            rm (list): List of entries to be removed
+            detach (bool): Detach flag
+            restart (str): Restart flag
+            cmds (str): Other commands to be executed
+        """
+        if not image and not rm:
+            raise ContainerError("Image name or rm option needs to be provided")
```

Comment:
instead use this exception - https://github.com/red-hat-storage/cephci/blob/master/cli/exceptions.py#L7